api/src/main/scala/hmda/api/model/SingleValidationErrorResult.scala

```diff
@@ -0,0 +1,10 @@
+package hmda.api.model
+
+import hmda.validation.engine.ValidationErrorsSummary
+
+case class SingleValidationErrorResult(
+  syntactical: ValidationErrorsSummary,
+  validity: ValidationErrorsSummary,
+  quality: ValidationErrorsSummary,
+  `macro`: ValidationErrorsSummary
```

Comment:
Do we want to have macro errors in this object? To my understanding they can't be run on a single Lar.